WebRoot/js/ajax/dwt/widgets/DwtCheckbox.js

```diff
@@ -317,13 +336,23 @@ function(templateId, data) {
     data.value = this._initValue;
 	data.type = this.INPUT_TYPE;
 	DwtControl.prototype._createHtmlFromTemplate.call(this, templateId, data);
+	//set custom style for check box only
+	this.CUSTOM_CHECKBOX_STYLE = DwtCheckbox.CUSTOM_CHECKBOX && this.INPUT_TYPE == 'checkbox';
```

Comment:
Just to confirm: DwtCheckbox.CUSTOM_CHECKBOX  will always be true as we set it true as default. Should this be this.CUSTOM_CHECKBOX for any child widget inheriting this to disabled the custom behavior?